src/libraries/Microsoft.Diagnostics.Tracing.EventSource.Redist/src/Microsoft/Diagnostics/Tracing/StubEnvironment.cs

```diff
@@ -59,16 +59,3 @@ public static int TrailingZeroCount(uint value)
         }
     }
 }
-
-internal static partial class Interop
-{
-    [SuppressUnmanagedCodeSecurityAttribute]
```

Comment:
The shared copy of Interop files does not have `SuppressUnmanagedCodeSecurityAttribute`. Is this refactoring going to regress performance on .NET Framework, in some cases at least?

Note that `Microsoft.Diagnostics.Tracing.EventSource.Redist` targets .NET Framework.

Reply:
Hi @jkotas, thanks for the review. Should I add the SuppressUnmanagedCodeSecurityAttribute to the shared copy of GetCurrentProcessId and GetCurrentThreadId ?

Reply:
I would keep it in non-shared partial class that is specific to `Microsoft.Diagnostics.Tracing.EventSource.Redist` project. We do not need the `SuppressUnmanagedCodeSecurityAttribute` annotations for non-.NET Framework projects that use the shared interop definitions under Common.

Reply:
cc: @brianrob, @noahfalk, as we were just talking about continued maintenance of this OOB.

Reply:
I understand. If it's OK for you, I will close this PR since there's no other refacto to do for Microsoft.Diagnostics.Tracing.EventSource.Redist library